lavalink/nodemanager.py

```diff
@@ -100,9 +116,24 @@ def find_ideal_node(self, region: str = None):
         best_node = self.find_ideal_node(node.region)
 
         if not best_node:
-            # TODO: Move players to a queue, wait for a node to connect?
-            log.error('Unable to move players, no available nodes!')
+            for player in node.players:
+                self.player_queue.append(player)
```

Comment:
You can use `self.player_queue.extend(node.players)` to add all of the node's players to the queue. Not sure which method is more performant so I'm open to keeping it this way if it's faster.